src/main/scala/viper/gobra/Gobra.scala

```diff
@@ -143,6 +165,13 @@ trait GoVerifier extends StrictLogging {
     if (allErrors.isEmpty) VerifierResult.Success else VerifierResult.Failure(allErrors)
   }
 
+  private def createSubmitter(allowSubmission: Boolean, id: String, verifier: String): Unit = {
```

Comment:
we should make the signature of `createSubmitter` by having `verifier` of type `ViperBackend`. We could obtain a the name using the `name` field that I suggested.